lms/envs/aws.py

```diff
@@ -696,6 +705,9 @@
 FIELD_OVERRIDE_PROVIDERS += (
```

Comment:
@rlucioni: I'd encourage you to make your wrapper work correctly with the bulk_operations functionality, if possibl.